test/teaspoon_env.rb

```diff
@@ -13,7 +13,7 @@
 
   config.suite do |suite|
     suite.use_framework :mocha
-    suite.matcher = "{test/javascripts,app/assets}/**/*_test.{js,js.coffee,coffee,coffee.erb}"
+    suite.matcher = "{test/javascripts,app/assets}/**/*_test.js"
```

Comment:
```suggestion
    suite.matcher = "{test/javascripts,app/assets}/**/*Test.js"
```
I would say using camelCase is more of a JS standard, but this is a nitpick.